vllm/worker/neuron_model_runner.py

```diff
@@ -245,6 +292,9 @@ def execute_model(
             raise ValueError(
                 "NeuronModelRunner does not support multi-step execution.")
 
+        # Update Neuron's generation configs from sampling_metadata
```

Comment:
nit. the comment doesn't add anything beyond what the method name already says

Reply:
removed.